ofonotextchannel.cpp

```diff
@@ -479,6 +480,7 @@ void oFonoTextChannel::mmsReceived(const QString &id, uint handle, const QVarian
     header["message-token"] = QDBusVariant(id);
     header["message-sender"] = QDBusVariant(handle);
     header["message-received"] = QDBusVariant(QDateTime::currentDateTimeUtc().toTime_t());
+    header["message-sent"] = QDBusVariant(QDateTime::fromString(info["SentTime"].toString(), Qt::ISODate).toTime_t());
```

Comment:
You need to define the variable info here, otherwise:
`error: 'info' was not declared in this scope`

Reply:
Basically that means that info must be filled first from the data like it is done for the SMS

Reply:
oups sorry